custom_components/bambu_lab/coordinator.py

```diff
@@ -192,7 +192,7 @@ def _update_print_error(self):
             }
             #LOGGER.debug(f"EVENT: print_error cleared: {event_data}")
             if 'Code' in device.print_error.errors:
```

Comment:
```suggestion
            if 'Code' in device.print_error.error:
```
This also needs to be changed

Reply:
Yes, you are right, I just noticed it by running it on my HA instance